interim-jan-2021/resolver-behavior-self-description.md

```diff
@@ -0,0 +1,27 @@
+# Resolver behavior self-description
+
+## Defining local-only namespaces
```

Comment:
I think we ought to have a requirement that the protocol enables clients to check that the local namespace is authorized, if it collides with a global namespace.

Reply:
Good point. I'll address with an update.